source/js/buyers-guide/newsletter-box.js

```diff
@@ -5,12 +5,14 @@
  */
 const NewsletterBox = {
   toggleVisibilityClasses: () => {
-    const buyersGuideNewsletterBox = document.querySelector(
```

Comment:
There can be multiple newsletter boxes on the same page. Changing this to use class instead of id